d2l-icon.html

```diff
@@ -92,6 +103,7 @@
 
 			__onChange: function() {
 				this.__initialize();
+				this.updateStyles(); // need to re-calc iron-icon variables
```

Comment:
I'm not sure why this is necessary. You aren't setting the `iron-icon` variables in the Javascript?